homeassistant/components/apple_tv.py

```diff
@@ -91,7 +91,7 @@ def configuration_callback(callback_data):
         hass.async_add_job(configurator.request_done, instance)
 
     instance = configurator.request_config(
-        hass, 'Apple TV Authentication', configuration_callback,
+        'Apple TV Authentication', configuration_callback,
```

Comment:
Ah, right. Then I'll just sit tight. Would be great if it was included in next minor!